### ISSUE TITLE
automountd fails to start on boot up

### ISSUE THREAD

**Issue by boilednut** [trueos/trueos-core#1452]
On OS boot up, as shown in the logs below, automountd is failing to start due to /dev/autofs not yet being available. Shortly after boot, automountd can be started properly. This issue has been occurring consistently since the update in June.

Jun 23 12:55:14  automountd[1709]: failed to open /dev/autofs:
 No such file or directory
Jun 23 12:55:14  /etc/init.d/automountd[1706]: start-stop-daem
on: failed to start `/usr/sbin/automountd'
Jun 23 12:55:14  /etc/init.d/automountd[1694]: ERROR: automoun
td failed to start
Jun 23 12:55:14  /etc/init.d/automount[1712]: ERROR: cannot st
art automount as automountd would not start

**Comment by Mrt134** [comment]
Thanks for reporting this. We'll take a look soon.

**Comment by beanpole135** [comment]
After additional followup with the OP on Discourse[1], it was determined that the autofs package was not installed. Installing the relevant package(s) resolved the issue.

[1]https://discourse.trueos.org/t/automount-and-autounmount-services-dont-launch-at-boot/1719/12

**Comment by boilednut** [comment]
My issue wasn't the same issue discussed in the referenced Discourse thread -- as the following point illustrates:

- As can been seen in the logs I included, automount was failing to start because a dynamically-created device file (/dev/autofs) wasn't available at startup -- not because the automount-related executables weren't installed. The autofs package is installed on the machine -- which is confirmed by the fact that the automount services were manually started successfully after boot. My issue appears to be due to a race condition at startup.

However, at this point, I'm no longer able to reproduce the problem. I've had a recent system update; so, perhaps, the problem was resolved in it.

If the problem reoccurs, I'll recreate this ticket.
